### ISSUE TITLE
Folder icons being replaced by MEGA logo

### ISSUE THREAD

**Issue by da2x** [meganz/MEGAsync#121]
Nautilus and Dolphin file managers show a huge round MEGA icon instead of the regular folder icons for folders I’ve synced to MEGA. This is MEGA ugly. How can I stop MEGA from messing with my icons?

I’ve disabled the Settings: Advanced: Folder overlay icon setting, but not sure if this is even relevant.

This issue only affects folders I’ve selected for synchronization, and not subfolder within those folders.

GNOME 3.24.2.1, Fedora 26, MEGAsync 3.1.4

**Comment by polmr** [comment]
Overlay icons are the small icons that indicate the status of the folders/files located within a synched folder.  To change the icon back to the default folder icon in Fedora 26, open your file browser, right click on the folder -> Properties and then click on the M icon and chose "Revert"

**Comment by tkuraku** [comment]
I agree. There should be an option in the application to not have the folder icons changed.

**Comment by carlos-rpg** [comment]
It is indeed very unpleasant to look at, and confusing; they don't look like folders anymore.

**Comment by Barraguesh** [comment]
So it will never change or what?

**Comment by Salah-Tabet** [comment]
Just In case someone is looking for the answer. The solution is simple, you just have to right-click on the folder go to properties-->Customise -> change icon, scroll down until you find the regular folder icon, click on it. apply then save.  hopefully, that helped. That was just a wat to mark the folder as successfully uploaded to the Mega Cloud.

**Comment by FGD-Garuda** [comment]
5 years later.

I still get the MEGA icon on all folders and files synced to MEGAsync.
This is with KDE 5.24.3.
Packages tried are `dolphin-megasync`, `dolphin-megasync-bin`, `dolphin-megasync-git`, on Arch.

![MEGA1](https://user-images.githubusercontent.com/35015576/160297136-88cf767f-4b38-40f6-9118-0c8258cdd84f.png)

![MEGA2](https://user-images.githubusercontent.com/35015576/160297123-73fa52bd-3490-413a-922e-bac1c558d3f0.png)


But the correct overlay icons do exist:

![MEGA3](https://user-images.githubusercontent.com/35015576/160297185-fa15d49e-1e48-4578-8840-eb28c8145801.png)

So how can we make reference to them (first 3 icons from the left in above pic)?


Thanks

**Comment by FGD-Garuda** [comment]
Hi!

> for folder in */; do gvfs-set-attribute -t unset "$folder" metadata::custom-icon; done

`gvfs-set-attribute` does not seem to exist in Arch, but your script and explanation got me thinking and I think I know what to do.

Thank a lot! 😃

**Comment by tfysekis** [comment]
> Hi!
> 
> > for folder in */; do gvfs-set-attribute -t unset "$folder" metadata::custom-icon; done
> 
> `gvfs-set-attribute` does not seem to exist in Arch, but your script and explanation got me thinking and I think I know what to do.
> 
> Thank a lot! smiley

Glad i gave you an idea,

it's possible that your unable to execute the command because `gvfs` is not installed on their system. 
You can check if `gvfs` is installed by running the command 
```
`gvfs-info --version`
```
Good luck!!
